sdk/worker.go

```diff
@@ -52,8 +66,8 @@ type Model struct {
 	UserLastModified time.Time          `json:"user_last_modified"  db:"user_last_modified"`
 	CreatedBy        User               `json:"created_by" db:"-"`
 	Provision        int64              `json:"provision" db:"provision"`
-	OwnerID          int64              `json:"owner_id" db:"owner_id"` //DEPRECATED
 	GroupID          int64              `json:"group_id" db:"group_id"`
+	Group            Group              `json:"group" db:"-"`
```

Comment:
By adding this, you must be sure to always return the Group for each Model in all CDS API